x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/select_type.test.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+
+import { NetworkTopNFlowDirection, NetworkTopNFlowType } from '../../../../graphql/types';
+
+import { NetworkTopNFlowTableId } from '.';
+import { SelectType } from './select_type';
+
+describe('NetworkTopNFlow Select direction', () => {
+  const mockOnChange = jest.fn();
+
+  describe('rendering', () => {
+    test('it renders the default Authentication table', () => {
```

Comment:
Yep, this should read `Network` and not `Authentication` table.

It is official! The snapshot test diffs are useful.